src/views/Login.vue

```diff
@@ -1,57 +1,101 @@
 <template>
   <div class="container">
     <img src="../assets/wolox-logo.png" class="logo">
-    <div class="input-container">
-      <label for="name" class="label">First name</label>
-      <input id="name" class="input" v-model="firstName">
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Last name</label>
-      <input id="name" class="input" v-model="lastName">
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Email</label>
-      <input id="name" class="input" v-model="email">
-    </div>
-    <div class="input-container">
-      <label for="name" class="label">Password</label>
-      <input id="name" class="input" v-model="password" type="password">
-    </div>
-    <button :class="'primaryButton'" @click="printForm">Sign up</button>
+    <form @submit.prevent="onSubmit">
+      <div class="input-container">
+        <label for="name" class="label">First name</label>
+        <input id="firstName" v-model="firstName" class="input">
+      </div>
+      <div class="input-container">
+        <label for="lastName" class="label">Last name</label>
+        <input id="lastName" v-model="lastName" class="input">
+      </div>
+      <div class="input-container">
+        <label for="email" class="label">Email</label>
+        <input
+          id="email"
+          v-model="email"
+          :class="['input', ($v.email.$invalid && submitted) || !$v.email.email ? 'error-input' : '']"
```

Comment:
`'error-input' : ($v.email.$invalid && submitted) || !$v.email.email)`

Reply:
why not only `'error-input' : ($v.email.$invalid && submitted)`?

Reply:
Because while it's being written and it doesn't have the format of an email, I want it to appear in red and `!$v.email.email` checks that =)

Reply:
Ok, but if I do what you say, I will have to add `'input': true`